packages/commercetools/theme/components/AppHeader.vue

```diff
@@ -0,0 +1,232 @@
+<template>
+  <div>
+    <SfOverlay :visible="!!hovered" />
+    <SfHeader
+      data-cy="app-header"
+      @click:cart="toggleCartSidebar"
+      @click:wishlist="toggleWishlistSidebar"
+      @click:account="handleAccountClick"
+      @enter:search="changeSearchTerm"
+      @change:search="p => term = p"
+      :searchValue="term"
+      :cartItemsQty="cartTotalItems"
+      :accountIcon="accountIcon"
+      class="sf-header--has-mobile-search"
+    >
+      <!-- TODO: add mobile view buttons after SFUI team PR -->
+      <template #logo>
+        <nuxt-link data-cy="app-header-url_logo" :to="localePath('/')" class="sf-header__logo">
+          <SfImage src="/icons/logo.svg" alt="Vue Storefront Next" class="sf-header__logo-image"/>
+        </nuxt-link>
+      </template>
+      <template #navigation>
```

Comment:
you only need a navigation slot and nothing else, please create a component eg. `TopMenu` or something like this in the core theme and then override only this in the CT